tests/e2e/vppagent.go

```diff
@@ -15,36 +15,58 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
 	"path/filepath"
 	"regexp"
 	"strings"
 	"testing"
 
-	"github.com/go-errors/errors"
-
 	docker "github.com/fsouza/go-dockerclient"
+	"github.com/go-errors/errors"
 	"github.com/vishvananda/netns"
+	"go.ligato.io/cn-infra/v2/health/statuscheck/model/status"
 	"go.ligato.io/cn-infra/v2/logging"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/proto"
+
+	"go.ligato.io/vpp-agent/v3/client"
+	"go.ligato.io/vpp-agent/v3/cmd/agentctl/api/types"
+	ctl "go.ligato.io/vpp-agent/v3/cmd/agentctl/client"
+	"go.ligato.io/vpp-agent/v3/pkg/models"
+	kvs "go.ligato.io/vpp-agent/v3/plugins/kvscheduler/api"
 	"go.ligato.io/vpp-agent/v3/plugins/linux/ifplugin/linuxcalls"
+	"go.ligato.io/vpp-agent/v3/proto/ligato/kvscheduler"
+
+	. "github.com/onsi/gomega"
 )
 
 const (
-	vppAgentLabelKey               = "e2e.test.vppagent"
-	vppAgentNamePrefix             = "e2e-test-vppagent-"
-	defaultStopContainerTimeoutSec = 3
+	agentImage       = "ligato/vpp-agent:latest"
+	agentLabelKey    = "e2e.test.vppagent"
+	agentNamePrefix  = "e2e-test-vppagent-"
+	agentInitTimeout = 15 // seconds
+	agentStopTimeout = 3  // seconds
 )
 
 var vppPingRegexp = regexp.MustCompile("Statistics: ([0-9]+) sent, ([0-9]+) received, ([0-9]+)% packet loss")
 
 // Agent represents running VPP-Agent test component
 type Agent struct {
 	ComponentRuntime
-	ctx  *TestCtx
-	name string
+	Client ctl.APIClient
```

Comment:
Exported field again. Is it needed? Definitely not now and i don't see the future changes as there is no additional PR for them. So no visible need to make it exported in this PR.